apps/odo/.formatter.exs

```diff
@@ -0,0 +1,5 @@
+# Used by "mix format"
```

Comment:
do we have a formatter for each app in the apps directory or can we get away with a single top-level one?

Reply:
We have one per app. Not sure if we can do a top level one or not.

Reply:
we should experiment if those different files will conflict with each other. If they have different values (like line length) and you run the mix command at one level, i suspect you could be undoing the work the formatter did when you ran it at the top level and vice versa.